DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/SelfAssessmentOverview.cshtml

```diff
@@ -16,7 +16,7 @@
     .Max();
   var competencySummaries = from g in Model.CompetencyGroups
                             let questions = g.SelectMany(c => c.AssessmentQuestions).Where(q => q.Required)
-                            let selfAssessedCount = questions.Count(q => q.ResultId.HasValue)
+                            let selfAssessedCount = questions.Count(q => q.Result == 1)

```

Comment:
Not convinced this is correct, @rshrirohit. We should count if Result has any value that is not null.

Reply:
Hi, the 'Result' may contain the following values - 
0- No
1- Yes, Achieved, Not Applicable
2- Practising

if we consider any value that is not null, we will include the 0 and 2 as well which won't be the correct count since they depict 'not achieved' or 'practising'

Reply:
@rshrirohit, a learner making a self assessment that is "Not achieved" or "Practising" is still a self assessment (they are self assessing that they have not yet met the competency requirement) and should be included in the count in this scenario, please.

Reply:
@kevwhitt-hee completed the changes as per the review comments. Please check.